modified_files/net/net.gypi

```diff
@@ -817,6 +817,8 @@
       'http2/hpack/decoder/hpack_block_decoder.h',
       'http2/hpack/decoder/hpack_decoder_string_buffer.cc',
       'http2/hpack/decoder/hpack_decoder_string_buffer.h',
+#      'http2/hpack/decoder/hpack_decoder_tables.cc',
```

Comment:
The script adds all files to the net.gypi files to minimize diffs and comments new ones out to minimize build size. Do not remove files from net.gypi manually -- they'll get added the next time the update script is run. If they aren't necessary for QUIC, then leave them commented. I think we should leave the HPACK files uncommented. I'm suprrised it's not being used anywhere ... isn't there a test anywhere that uses it?